e2e-tests/playwright/e2e/catalog-timestamp.spec.ts

```diff
@@ -41,6 +41,7 @@ test.describe('Test timestamp column on Catalog', () => {
     await uiHelper.verifyRowInTableByUniqueText('timestamp-test', [
       /^\d{1,2}\/\d{1,2}\/\d{1,4}, \d:\d{1,2}:\d{1,2} (AM|PM)$/g,
     ]);
+    await uiHelper.searchInputPlaceholder('');
```

Comment:
we can again go into issue if all the row in catalog page have CreatedAt value.
instead of clearing the search I would say lets add a component without CreatedAt value which have similar name to the one used in this test. for eg: `timestamp-test-2` and do the partial search `timestamp-test` so both components will be visible and toggle assertion should work as expected.

Reply:
Ok, agreed. It will be more bullet-proof your way 😄

Reply:
Updated https://github.com/janus-qe/custom-catalog-entities/blob/main/timestamp-catalog-info.yaml and closing this PR.

Reply:
We are gonna need to update the test anyway. The `timestamp-test` contained the `Created at` and it was at the first place (without sorting). And the second test requires it to be empty at the first place. So I updated the `timestamp-catalog-info.yaml` again and we need this PR to address that.